packages/llvm.rb

```diff
@@ -1,38 +1,56 @@
 require 'package'
 
 class Llvm < Package
-  description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies.'
-  homepage 'https://llvm.org/'
-  version '5.0.1'
-  source_url 'https://releases.llvm.org/5.0.1/llvm-5.0.1.src.tar.xz'
-  source_sha256 '5fa7489fc0225b11821cab0362f5813a05f2bcf2533e8a4ea9c9c860168807b0'
+    description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies.'
+  homepage 'https://compiler-rt.llvm.org/'
```

Comment:
Just curious, why did you change the homepage link?

Reply:
Originally, I was about to fix compiler-rt problem.